### ISSUE TITLE
Change display values

### ISSUE THREAD

**Issue by SirCameron** [plotly/react-plotly.js#144]
I have 2 plots displayed on one graph and I changed the values of one of the plots to ensure it was within the same y axis range as the other plot..
Is it possible to show different values on the value tooltips? 
I would just need to receive the value at the current position and run a calculation on it to get the original value..

**Comment by nicolaskruchten** [comment]
Yep, check out the “hovertemplate” attribute

**Comment by SirCameron** [comment]
Thanks. But, it looks like this is just a simple string template. No functionality to specify a function that returns augmented data.

**Comment by nicolaskruchten** [comment]
That’s right: Plotly.js figures are totally declarative and therefore accept no functions. That said, you can store derived data in “customdata” and refer to it in “hovertemplate”
